backend/src/main/resources/graphql/main.graphqls

```diff
@@ -365,8 +365,8 @@ type Query {
   ): [Facility]
   patients(
     facilityId: ID
-    pageNumber: Int
-    pageSize: Int
+    pageNumber: Int = 0
```

Comment:
out of curiosity how did we determine the defaults?

Reply:
we already had these values in the java as defaults 
https://github.com/CDCgov/prime-simplereport/blob/4ea880b87358608c21bd20bd21ec5512671b9837/backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java#L44-L45

https://github.com/CDCgov/prime-simplereport/blob/4ea880b87358608c21bd20bd21ec5512671b9837/backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java#L186-L199